stable/prometheus-operator/templates/grafana/servicemonitor.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if and .Values.grafana.enabled .Values.grafana.serviceMonitor.selfMonitor }}
+apiVersion: {{ printf "%s/v1" (.Values.prometheusOperator.crdApiGroup | default "monitoring.coreos.com") }}
+kind: ServiceMonitor
+metadata:
+  name: {{ template "prometheus-operator.fullname" . }}-grafana
+  labels:
+    app: {{ template "prometheus-operator.name" . }}-grafana
+{{ include "prometheus-operator.labels" . | indent 4 }}
+spec:
+  selector:
+    matchLabels:
+      app: grafana
+      release: {{ .Release.Name | quote }}
+  namespaceSelector:
+    matchNames:
+      - {{ .Release.Namespace | quote }}
+  endpoints:
+  - port: service
+    interval: 30s
```

Comment:
If this value is left unset, then the default becomes the prometheus default scrape interval. It's good to allow this to be overridden, but it's easier to reason about metrics if they are all collected using the same scrape interval.

Reply:
@vsliouniaev I don't mind removing the interval override of 30s, but both Prometheus and Alertmanager have the same override. Let me know what you prefer.

Reply:
Ah, didn't realize that was the case. It would make sense to pull this out into configuration settings then but I see no reason to amend this in your PR.